config/crd/bases/sriovnetwork.openshift.io_sriovibnetworks.yaml

```diff
@@ -1,10 +1,9 @@
-
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.9.0
```

Comment:
why was this change introduced ? (when bumping go 1.18 needed to bump contoller-gen kubebuilder ?)

Reply:
yep I was not able to make the controller-gen version 0.4.1 to work with golang 1.18 so I had to bump it